Dockerfile

```diff
@@ -34,7 +34,8 @@ RUN go mod download
 # cross-compilation setup
 ARG TARGETARCH
 RUN xx-go --wrap && \
-    GO_LDFLAGS="-linkmode=external -X github.com/k3s-io/k3s/pkg/version.Program=rke2 -X github.com/k3s-io/k3s/pkg/version.Version=$(go list -f '{{.Version}}' -m github.com/k3s-io/k3s)" \
+    GIT_COMMIT=$(git rev-parse --short HEAD) \
+    GO_LDFLAGS="-linkmode=external -X github.com/k3s-io/k3s/pkg/version.Program=rke2 -X ${PKG}/pkg/version.GitCommit=${GIT_COMMIT} -X github.com/k3s-io/k3s/pkg/version.Version=$(go list -f '{{.Version}}' -m github.com/k3s-io/k3s)" \
```

Comment:
this package (`github.com/rancher/image-build-rke2-cloud-provider`) does not have its own `pkg/version`. All it has is the main.go entrypoint; everything else is from k3s.

Reply:
yeah i figured as much :/ i was doing this so that we could pull the proper info to vex automatically, but this isn't the right way (unsurprisingly)